addons/web/static/src/js/chrome/search_view.js

```diff
@@ -625,6 +610,34 @@ var SearchView = Widget.extend({
         fv.arch = utils.xml_to_json(doc, true);
         return fv;
     },
+    _setSearchViewButtons: function() {
```

Comment:
I would like to avoid refactoring.
keep changes to a minimum in this file.
We will refactor this later.